src/downloads.jl

```diff
@@ -28,6 +28,7 @@ See Also
 """
 
 function yahoo(data::String="^GSPC")
+    Base.depwarn("Yahoo Finance API changed, this function may not work anymore", :yahoo)
```

Comment:
IIRC, yahoo API is shut down.

Reply:
Not so clear...
See `https://github.com/pydata/pandas-datareader/issues/549`